site/src/pages/UserSettingsPage/AccountPage/AccountPage.tsx

```diff
@@ -20,10 +21,13 @@ export const AccountPage: React.FC = () => {
   return (
     <Section title={Language.title}>
       <AccountForm
+        editable={!!canEditUsers}
```

Comment:
nit: `Boolean()` feels more readable than `!!`. Probably something that should be discussed and if agreed by team, enforced via ESLint [`no-implicit-coercion`](https://eslint.org/docs/latest/rules/no-implicit-coercion)
```suggestion
        editable={Boolean(canEditUsers)}
```

Reply:
Yeah I just got impatient here 😅

Reply:
Added to next FE Variety to discuss!

Reply:
Lol can't blame you!